integration-cli/docker_api_network_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/docker/docker/api"
+	"github.com/go-check/check"
+)
+
+func isNetworkAvailable(c *check.C, name string) bool {
+	status, body, err := sockRequest("GET", fmt.Sprintf("/v%s/networks", api.APIVERSION), nil)
```

Comment:
Is there some reason we are passing in an `api.APIVERSION` here? If this is not specified it should just default to whatever is defined here anyway.